src/main/steps/applicant1/hub-page/sole/content.ts

```diff
@@ -851,6 +881,20 @@ export const generateContent: TranslationFn = content => {
     userCase.applicant1NoResponsePartnerNewEmailOrPostalAddress ===
       NoResponsePartnerNewEmailOrPostalAddress.CONTACT_DETAILS_UPDATED &&
     userCase.applicant2AddressOverseas !== YesOrNo.YES;
+  const interimApplicationStartPagePath = (() => {
+    switch (userCase.applicant1InterimApplicationType) {
+      case InterimApplicationType.ALTERNATIVE_SERVICE:
+        return ALTERNATIVE_SERVICE_APPLICATION;
+      case InterimApplicationType.DEEMED_SERVICE:
+        return DEEMED_SERVICE_APPLICATION;
+      case InterimApplicationType.BAILIFF_SERVICE:
+      case InterimApplicationType.DISPENSE_WITH_SERVICE:
+      default: // Remove when all the options are completed
+        return HUB_PAGE;
+    }
+  })();
+  const interimApplicationStartedAosOverdue =
+    userCase.applicant1InterimApplicationType !== undefined && userCase.state === State.AosOverdue;
```

Comment:
I'd probably tweak this slightly to handle other unpopulated values for application type, such as null and empty strings:
```suggestion
   !userCase.applicant1InterimApplicationType && userCase.state === State.AosOverdue;
```